src/relay/backend/interpreter.cc

```diff
@@ -270,16 +270,30 @@ class Interpreter :
     return TupleValueNode::make(values);
   }
 
-  Value VisitExpr_(const FunctionNode* func_node) final {
-    auto func = GetRef<Function>(func_node);
+  // TODO(@jroesch): this doesn't support mututal letrec
```

Comment:
This diff is not necessary as it's already in mainline by https://github.com/dmlc/tvm/commit/2df3364b0594d15053e23cee038305a6b52f144e#diff-a90b7229a4e9fb69bb84bb8f75c162e8L273.